src/app/components/crypto-assets/components/asset-caption.tsx

```diff
@@ -17,21 +19,15 @@ export function AssetCaption({ caption, isUnanchored }: AssetCaptionProps) {
             • Microblock
           </styled.span>
           <Tooltip placement="right-end" label={'Learn more about microblocks'}>
-            <Stack isInline>
+            <HStack>
               <a
                 href="https://docs.stacks.co/understand-stacks/microblocks"
                 target="_blank"
                 rel="noreferrer"
               >
-                <Box
-                  _hover={{ cursor: 'pointer' }}
-                  size="12px"
-                  color={color('text-caption')}
-                  as={FiInfo}
-                  ml={1}
-                />
+                <InfoIcon size="12px" ml={1} color="accent.text-subdued" />
```

Comment:
Think `ml` needs a space token here?

Reply:
Thanks, I update it. I'll go through my other icons